tests/bulkdata/bulk_data_delete_tests.py

```diff
@@ -0,0 +1,73 @@
+from django.contrib.auth.models import User
+from arches.app.models.resource import Resource
+from arches.app.models.tile import Tile
+from arches.app.models import models
+from arches.app.utils.data_management.resource_graphs.importer import import_graph as ResourceGraphImporter
+from arches.app.utils.betterJSONSerializer import JSONSerializer, JSONDeserializer
+from yourapp.models import BulkDataDeletion  # Adjust import as necessary
+from tests.arches_test_case import ArchesTestCase
+import os
+import uuid
```

Comment:
We don't have a hook for isort, but I usually put stdlib imports first.

Reply:
fix in latest